TACOCAT/src/Flux_Profiles.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import matplotlib.pyplot as plt 
+import pandas as pd
+import math
+import os
+import sys
+sys.path.insert(0,'../..') #This adds the ability to pull Flux profiles from the main folder
+from pathlib import Path
+import TACOCAT_Read_In_File as TCinput
+
+steps = 36
+Hc = TCinput.Hc
+z = np.linspace(-Hc/2,Hc/2,steps)
+k = 1
+
+#Flat Line Flux Profile (x = 1)
+FlatLineFlux = np.zeros(steps)
+FlatLineFlux[:] = 1
+plt.figure(k)
```

Comment:
Can you remove the figures from this? We just want to make it so the user can use this flux profile in the future.